relnotes/resume-fiber.feature.md

```diff
@@ -0,0 +1,8 @@
+* `swarm.neo.connection.RequestOnConnBase`
+
+  The nested class `EventDispatcher` now has the ability to suspend the outer
```

Comment:
Strange how this talks about suspending the fiber, while this patch is adding `resumeFiber` method.